bigchaindb/tendermint/lib.py

```diff
@@ -353,10 +354,14 @@ def validate_transaction(self, tx, current_transactions=[]):
         return transaction.validate(self, current_transactions)
 
     def is_valid_transaction(self, tx, current_transactions=[]):
+        # The below exception handling is quite broad because this method
+        # is called by ABCI proxy app. An exception not caught in this method
+        # will cause a crash in the ABCI server leaving the system in a
```

Comment:
Under the circumstances, might makes sense to additionally catch all the exceptions here to be on the safe side. If it's not `BigchainBDError`, log an error and return False, I suppose?

Reply:
As a general practice suggested by @vrde only expected exceptions should be handeled. Since `BigchainDBError` is the base class for all BigchainDB specific exceptions, so that has been handled here.

@vrde Any thoughts?